src/BenchmarksApps/TechEmpower/Minimal/Database/Db.cs

```diff
@@ -2,118 +2,117 @@
+        _connectionString = appSettings.ConnectionString;
+    }
 
-        private readonly DbProviderFactory _dbProviderFactory;
-        private readonly string _connectionString;
+    public Task<World> LoadSingleQueryRow()
+    {
+        using var db = _dbProviderFactory.CreateConnection();
+        db!.ConnectionString = _connectionString;
 
-        public Db(AppSettings appSettings)
-        {
-            ArgumentNullException.ThrowIfNull(appSettings.ConnectionString);
+        // Note: Don't need to open connection if only doing one thing; let dapper do it
+        return ReadSingleRow(db);
+    }
 
-            _dbProviderFactory = Npgsql.NpgsqlFactory.Instance;
-            _connectionString = appSettings.ConnectionString;
-        }
+    static Task<World> ReadSingleRow(DbConnection db)
+    {
+        return db.QueryFirstOrDefaultAsync<World>(
+                "SELECT id, randomnumber FROM world WHERE id = @Id",
+                new { Id = _random.Next(1, 10001) });
+    }
 
-        public Task<World> LoadSingleQueryRow()
+    public async Task<World[]> LoadMultipleQueriesRows(int count)
+    {
+        if (count <= 0)
         {
-            using var db = _dbProviderFactory.CreateConnection();
-            db!.ConnectionString = _connectionString;
-
-            // Note: Don't need to open connection if only doing one thing; let dapper do it
-            return ReadSingleRow(db);
+            count = 1;
+        }
+        else if (count > 500)
+        {
+            count = 500;
         }
 
-        static Task<World> ReadSingleRow(DbConnection db)
+        var results = new World[count];
+        using var db = _dbProviderFactory.CreateConnection();
+
+        db!.ConnectionString = _connectionString;
+        await db.OpenAsync();
+
+        for (var i = 0; i < count; i++)
         {
-            return db.QueryFirstOrDefaultAsync<World>(
-                    "SELECT id, randomnumber FROM world WHERE id = @Id",
-                    new { Id = _random.Next(1, 10001) });
+            results[i] = await ReadSingleRow(db);
         }
 
-        public async Task<World[]> LoadMultipleQueriesRows(int count)
+        return results;
+    }
+
+    public async Task<World[]> LoadMultipleUpdatesRows(int count)
+    {
+        if (count <= 0)
         {
-            if (count <= 0)
-            {
-                count = 1;
-            }
-            else if (count > 500)
-            {
-                count = 500;
-            }
-
-            var results = new World[count];
-            using var db = _dbProviderFactory.CreateConnection();
-
-            db!.ConnectionString = _connectionString;
-            await db.OpenAsync();
-
-            for (var i = 0; i < count; i++)
-            {
-                results[i] = await ReadSingleRow(db);
-            }
-
-            return results;
+            count = 1;
         }
+        else if (count > 500)
+        {
+            count = 500;
+        }
```

Comment:
You could use:

Math.Clamp(count, 1, 500);

Reply:
Again, not part of my change, but that's a nice API so will update to use it :smile: